seslib/__init__.py

```diff
@@ -491,7 +491,7 @@ def _generate_static_networks(self):
             self.settings.cluster_network = cluster_network
 
     def _generate_nodes(self):
-        node_id = 1
+        node_id = 0
```

Comment:
```suggestion
        node_id = 1
```
Either keep the id at 1 or increment it before it is used. Otherwise you get the naming issue you already marked, as well as using the 200 ip address twice.

Though actually that is also not ideal, as it keeps the loop counter increment somewhere in the loop where its easy to miss.

Reply:
Done. I'm now incrementing before using it.